File: Practice/08/python/08.py
chislo1,znak,chislo2=input().split() #ввод
if znak=='+':
    print(float(chislo1)+float(chislo2)) #сложение
if znak=='-':
    print(float(chislo1)-float(chislo2)) #вычитание
if znak=='*':
    print(float(chislo1)*float(chislo2)) #умножение
if znak=='/': #деление
    if chislo2!='0':
        print(float(chislo1)/float(chislo2))
    else:
        print('ошибка, нельзя на ноль делить')

File: Practice/08/c++/08/08.cpp
#include <iostream>
using namespace std;

int main()
{
    setlocale(LC_ALL, "RUSSIAN");
    double chislo1, chislo2;
    char znak;
    cout << "введите в следующей строке через пробел первое число знак и второе число\n";
    cin >> chislo1 >> znak >> chislo2;
    if (int(znak) == 43) cout << chislo1 + chislo2; //проверка знака по ascii коду
    if (int(znak) == 45) cout << chislo1 - chislo2;
    if (int(znak) == 42) cout << chislo1 * chislo2;
    if (int(znak) == 47) {
        if (chislo2 != 0) cout << chislo1 / chislo2;
        else cout << "нельзя делить на нол";
    }
}